src/olympia/addons/tests/test_views.py

```diff
@@ -1590,6 +1590,11 @@ def _test_url(self):
     def _set_tested_url(self, param):
         raise NotImplementedError
 
+    def _make_unlisted(self):
+        self.addon.update(is_listed=False)
+        for version in self.addon.versions.all():
+            version.update(channel=amo.RELEASE_CHANNEL_UNLISTED)
```

Comment:
Is this different than `self.addon.versions.update(channel=amo.RELEASE_CHANNEL_UNLISTED)`?